Resources/Textures/Structures/Wallmounts/switch.rsi/meta.json

```diff
@@ -4,8 +4,8 @@
     "x": 32,
     "y": 32
   },
-  "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from CEV-Eris at commit 15748b71a1e6d2c547588c653d7c42f18011f7c5",
+  "license": "CC0-1.0",
```

Comment:
wha

Reply:
Could you elaborate?

Reply:
anyway, these kinda look like derivatives of the old one so might be best to preserve the licensing and copyright.

Reply:
> anyway, these kinda look like derivatives of the old one so might be best to preserve the licensing and copyright.

Yeah you're completely right, fixed my mistake